contribution/vagrant/Vagrantfile

```diff
@@ -1,18 +1,27 @@
 Vagrant.require_version ">= 2.0.0"
 
+if ENV['NETNEXT'] == "1" then
+	# Using Cilium's Ubuntu Next image https://app.vagrantup.com/cilium/boxes/ubuntu-next
+	VM_IMG = "cilium/ubuntu-next"
+	VM_NAME = "kubearmor-dev-next"
+else
+	VM_IMG = "ubuntu/bionic64"
+	VM_NAME = "kubearmor-dev"
+end
+
 Vagrant.configure("2") do |config|
   config.vm.define "KubeArmor-Dev" do |cfg|
```

Comment:
I think that `config.vm.define` should be the same as `VM_NAME`, so the user can up and connect to both VMs interchangeably.

```diff
 Vagrant.configure("2") do |config|
-  config.vm.define "KubeArmor-Dev" do |cfg|
+  config.vm.define VM_NAME do |cfg|
```

If it is set statically, the right `VM_IMG` won't be up in despite of `NETNEXT` when other is already created.

Reply:
Handled, thanks